mod-template-engine/src/main/resources/domain/mod-template-engine/features/templates.feature

```diff
@@ -12,42 +14,106 @@ Feature: Templates tests
 
   # CRUD
 
-  @Undefined
   Scenario: Post should return 201 and new template
-    * print 'undefined'
+    * def requestEntity = read('samples/template-entity.json')
+
+    Given path 'templates'
+    And request requestEntity
+    When method POST
+    Then status 201
+
+    Given path 'templates/' + templateId
+    When method GET
+    Then status 200
+    And match $.description == requestEntity.description
 
-  @Undefined
   Scenario: Post should return 400 if template resolver is not supported
-    * print 'undefined'
+    * def requestEntity = read('samples/template-entity.json')
+    * requestEntity.templateResolver = {}
+
+    Given path 'templates'
+    And request requestEntity
+    When method POST
+    Then status 400
 
-  @Undefined
   Scenario: Post should return 422 if template did not pass validation
-    * print 'undefined'
+    * def requestEntity = read('samples/invalid-template-entity.json')
+
+    Given path 'templates'
+    And request requestEntity
+    When method POST
+    Then status 422
 
-  @Undefined
   Scenario: Get by id should return 200
-    * print 'undefined'
+    Given path 'templates'
+    When method GET
+    Then status 200
+
+    Given path 'templates/' + response.templates[0].id
+    When method GET
+    Then status 200
 
-  @Undefined
   Scenario: Get by id should return 404 if template does not exist
-    * print 'undefined'
+    Given path 'templates/' + templateNotFoundId
```

Comment:
you can use templateId. It is generated unique before each method, and it hasn't been added before